src/Panel/Panel.d.ts

```diff
@@ -0,0 +1,37 @@
+import { PureComponent, ReactNode } from 'react';
+
+export namespace PanelProps {
+
+  type HeaderSize = 'tiny' | 'small' | 'medium';
+
+  type CollapseDirection = 'up' | 'left' | 'down' | 'right';
+
+  type Header = {
+    collapse?: {
+      direction: CollapseDirection,
+      onExpand: () => void,
+      onCollapse: () => void,
+      isCollapsed?: boolean
+    },
+    size?: HeaderSize,
+    content?: any,
+    title?: ReactNode | string, // TODO(typo): wtf
```

Comment:
nope, `ReactNode` doesn't accept a `string`
```
type ReactNode = ReactChild | ReactFragment | boolean | null | undefined;
```

but I'm going to set to `any` until we'll find a right type for `children`